CSS/product_page.css

```diff
@@ -0,0 +1,190 @@
+@import url(/CSS/style.css);
+@import url(/CSS/wishlist.css);
+@import url(/CSS/cart.css);
+
+.product_card_list {
+    background-color: var(--background-color);
+    margin: 1rem;
+}
+
+.all_cards_container,
+.product_title,
+.landscap_card_container,
+.landscap_cards,
+.price_info,
+.landscap_btns,
+.price_range,
+.product_card_details,
+.price_list,
+.fillter_container,
+.product_container,
+.product_wishlist_icon,
+.containt_list {
```

Comment:
Create a utility class for this property and use that everywhere

Reply:
okay thank you , i will use